.gitignore

```diff
@@ -26,6 +26,7 @@ osx-nginx-ingress
 nginx-ingress
 osx-nginx-plus-ingress
 nginx-plus-ingress
+nginx-controller/nginx-controller
```

Comment:
what is this for?

Reply:
It's to ignore `nginx-controller` binary which appears when you run `go build` inside `nginx-controller` directory.